clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1935,6 +1935,26 @@ pointer address bits to bank on.
   }];
 }
 
+def IntelFPGAForcePow2DepthAttrDocs : Documentation {
+  let Category = DocCatVariable;
+  let Heading = "force_pow2_depth (IntelFPGA)";
+  let Content = [{
+This attribute may be attached to a variable or struct member declaration and
+provides explicit control over the geometry of memory blocks used in a given
+memory system.
+
+In the presence of this attribute, the compiler will:
+
+1. Automatically size the memory depth to the next largest power of 2 if
+force_pow2_depth is set to 1, and will prefer width-stitching of RAM blocks
+over depth-stitching.
+
+2. Will not size the memory to the next largest power of 2 if force_pow2_depth
+is set to 0, and will prefer depth stitching instead of width-stitching if RAM
```

Comment:
```suggestion
is set to 0, and will prefer depth-stitching over width-stitching if RAM
```

Reply:
Thank you for writing some documentation.
Probably this requires some more context to be understandable...
Any link to some external documentation?

Reply:
@keryell Actually I copied this description from code of my colleague. And I could find any documentation regarding this attribute. Maybe @GarveyJoe may help you.

Reply:
It's a fresh-new attribute and AFAIK there is no external documentation yet. But I believe, at some point it will appear, for example, here: https://www.intel.com/content/www/us/en/programmable/documentation/nml1505158467345.html#kzp1508549429833